### PR TITLE
Problem, Record and Patient Record Elasticsearch with testing complete

### PR DESCRIPTION
**Pull request by modelarious**
Had to open #199

### CODE REVIEW

**Comment by modelarious** [comment]
Addresses part of #172

File: app/src/main/java/Controllers/ElasticsearchProblemController.java [hunk 1]
```diff
@@ -63,79 +48,169 @@ public static void setClient(){
         }
     }
 
-    public static class DeleteAllProblemsForUserIDsTask extends AsyncTask<String, Void, Boolean>{
```
Comment (tnguyen10) [inline app/src/main/java/Controllers/ElasticsearchProblemController.java]: should we still have DeleteAllProblemsForUserIDsTask that deletes all problem associated to a user (calls when user is deleted) or should we just combine DeleteProblemsTask and GetProblemsCreatedByUserIDTask?
Also need to consider the extension to records

Comment (modelarious) [reply app/src/main/java/Controllers/ElasticsearchProblemController.java]: I totally missed that I had gotten rid of this! I'll add it back in.  I'll test it on a later pull request.

Comment (modelarious) [reply app/src/main/java/Controllers/ElasticsearchProblemController.java]: This has been added in as of 5a69680

**Comment by modelarious** [comment]
blocked by #218

**Comment by modelarious** [comment]
Addressed #172

**Pull request merged**